constellations/orion/src/router/handler.go

```diff
@@ -109,4 +109,14 @@ func (h *Handler) SetupApiEndpoints() {
 		apiAFH.DELETE("/afh/:id", controllers.DeleteAFH)
 	}
 	// apiAccounts := router.Group("api/accounts/")
+
+	apiUserAfh := h.Engine.Group("api/userAfhs")
```

Comment:
so turns out URLs are not case sensitive, to its the same as `api/userafhs`
As long as this is unique from the other above endpoints, I think it's ok to just keep this `api/userafhs`